xbe/__init__.py

```diff
@@ -479,11 +479,16 @@ class XbeImageHeader(ctypes.LittleEndianStructure, StructurePrintMixin):
 		('logo_size',                   ctypes.c_uint32),
 		]
 
-class XbeImageHeaderExtended(XbeImageHeader):
+class XbeImageHeaderExtended1(XbeImageHeader):
```

Comment:
Just a minor nit. Can you make this 'Type1' instead of '1' so it's clear that 2 does not follow 1, but is instead an alternative

Reply:
Done